examples/helloworld-cdi2-se/src/main/java/org/glassfish/jersey/examples/helloworld/cdi2se/App.java

```diff
@@ -25,7 +25,7 @@
  */
 public class App {
 
-    private static final URI BASE_URI = URI.create("http://localhost:8080/base/");
+    private static final URI BASE_URI = URI.create("http://localhost:8080/");
     public static final String ROOT_HELLO_PATH = "helloworld";
```

Comment:
I would recommend to add '/Some Name' here as well, it misleads otherwise

Reply:
That makes no sense. "Some Name" is not part of the application's path, it is just a demo value to make the link contained in the readme actually work, otherwise the application fails due to missing input, as hello world requests that input value.

Reply:
actually it does make sense :) When you run the example, it shows the output:
```
Application started.
Try out
http://localhost:8080/helloworld
http://localhost:8080/counter/request
http://localhost:8080/counter/application
```

and when you use ```http://localhost:8080/helloworld``` link, it gives you ```HTTP ERROR 404```.

But as soon as you use ```http://localhost:8080/helloworld/some%20name``` you get a page with

> Hello some name

text on it

I do not say the the exactly _some name_ text shall be used :) it was just an example value. But if you put it to README, why not to put it to server's output? Other links in that output are perfectly valid.

I would say ```http://localhost:8080/helloworld/{input} would be saying to user that there should be something after helloworld rather then just http://localhost:8080/helloworld which leads to 404. 

but as soon as **ROOT_HELLO_PATH** variable is used inside a test, it could be added directly to server's output ```System.out.println(String.format("%s%s%s", BASE_URI, ROOT_HELLO_PATH, "/{input}"));```

Reply:
Ah, you want to add it to `ROOT_HELLO_PATH`! This makes sense. In fact I assumed you wanted to add it to`BASE_URI`, which does not make sense. Sure, I will change that. Meanwhile I hope Travis CI will fix their steadily failing infrastructure... ;-)